torch/_inductor/scheduler.py

```diff
@@ -986,6 +1064,11 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode):
         ):
             return False
 
+        if isinstance(node1, ForeachKernelSchedulerNode) ^ isinstance(
```

Comment:
` or `

Reply:
I want it to continue if they're both ForeachKernelSchedulerNodes though

Reply:
Perhaps `is not`